services.html

```diff
@@ -0,0 +1,84 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="utf-8"
+        content="width=device-width, initial-scale=1"
+        name="viewport">
+  <title>Great Idea!</title>
+  <link href="https://fonts.googleapis.com/css?family=Bangers|Titillium+Web"
+        rel="stylesheet">
+  <link href="css/index.css"
+        rel="stylesheet"><!--[if lt IE 9]>
+      <script src="https://cdnjs.cloudflare.com/ajax/libs/html5shiv/3.7.3/html5shiv.js"></script>
+    <![endif]-->
+</head>
+<body>
+  <div class="container">
+    <header>
+      <nav>
+        <a href="services.html">Services</a> <a href="#">Product</a> <a href="#">Vision</a> <a href="#">Features</a> <a href="about.html">About</a> <a href="#">Contact</a>
```

Comment:
Typically it's easier to follow if you spread out your code like you would a webpage; you want it to be more skinny than wide (at least if you have a lot of content). Try to format your HTML more like this:
```html
<nav>
    <a href="services.html">Services</a>
    <a href="#">Product</a>
    <a href="#">Vision</a>
    <a href="#">Features</a>
    <a href="about.html">About</a>
    <a href="#">Contact</a>
</nav>
```
Giving each tag its own line is typical unless it's very small.
Do this for the future developer! Not everyone has a big screen, and even if they use wrap in their text editor, it'll still most likely make it harder to read!